src/components/MainView.js

```diff
@@ -33,19 +33,20 @@ export default class MainView extends React.Component {
   }
 
   export(json) {
-    let csv  = Papa.unparse(json),
-        blob = new Blob([csv], {type: 'text/csv'}),
-        url  = window.URL.createObjectURL(blob),
-        link = document.createElement('a'),
-        evt  = document.createEvent('MouseEvents');
-
-    link.setAttribute('href', url);
-    link.setAttribute('target', '_blank');
-    link.setAttribute('download', 'data.csv');
-
-    evt.initMouseEvent('click', true, true, document.defaultView, 1, 0, 0, 0, 0,
-      false, false, false, false, 0, null);
-    link.dispatchEvent(evt);
+    var filename = "data.csv";
+    var csv = Papa.unparse(json);
+    var blob = new Blob([csv], {type: 'text/csv'});
+    if (window.navigator.msSaveOrOpenBlob)  // IE hack; see http://msdn.microsoft.com/en-us/library/ie/hh779016.aspx
```

Comment:
I believe I already did add an eslint file. I have an eslint package in atom, but we should add a script to run it for those who don't.